### ISSUE TITLE
Dependency coffee-script triggers a failure on Windows.

### ISSUE THREAD

**Issue by Kylir** [mhevery/jasmine-node#422]
Hi All,

There is a bug in the latest version of `coffee-script` (one of the dependencies of `jasmine-node`.)
It's post-install script is making the `npm install` fail.

See here: https://github.com/jashkenas/coffeescript/issues/4805

Possible fix: upgrade the dependency to `coffeescript`?

**Comment by cha0s** [comment]
Yep, this should be done. I also just chased an error due to requiring the (now) correct `coffeescript` module and expecting it to be the same reference.
